airflow/www/views.py

```diff
@@ -3867,13 +3870,15 @@ class VariableModelView(AirflowModelView):
         'add': 'create',
         'list': 'read',
         'edit': 'edit',
+        'show': 'show',
```

Comment:
```suggestion
        'show': 'read',
```

List and get should be mapped to the same permission.

Reply:
done